partner-chains-cli/src/config.rs

```diff
@@ -376,11 +377,18 @@ pub struct SidechainParams {
 	pub governance_authority: MainchainAddressHash,
 }
 
+#[derive(Deserialize, Serialize, Parser, Clone, Debug)]
+pub struct NativeTokenConfig {
+	native_token_policy: String,
+	native_token_asset_name: String,
+}
+
 #[derive(Deserialize)]
 pub struct ChainConfig {
 	pub cardano: CardanoParameters,
 	pub chain_parameters: SidechainParams,
 	pub cardano_addresses: MainChainAddresses,
+	pub native_token: NativeTokenConfig,
```

Comment:
A: Is it already approved that we support using exactly one ?

B: this is logically bound to `illiquid_supply_address` from `cardano_addresses`. I think it is better to keep this asset id where we have other policy ids. And what is more, if not structurally bound like:
```
struct IlliquidSupplyConfig {
  asset: AssetConfig, // or assets: Vec[AssetConfig] - depends on decision how many are supported.
  address: String,
}
```
then at least name here should help to bind these together

Reply:
A: Yes, we're supporting only one until MN requires us to support multiple. That's decided by @gilligan 

B: Okay, I moved it into `cardano_addresses` under `native_token`